src/main/java/org/folio/innreach/domain/service/impl/InnReachTransactionActionServiceImpl.java

```diff
@@ -133,6 +137,23 @@ public void handleLoanUpdate(LoanDTO loan) {
         reportItemInTransit(transaction);
       });
     }
+
+    if (loan.getAction().equals("renewed")) {
+      var folioLoanId = loan.getId();
+
+      transactionRepository.fetchOneByLoanId(folioLoanId).ifPresent(transaction -> {
+        log.info("Loan with id: {} associated with transaction. Transaction id: {}", folioLoanId, transaction.getId());
+        var transactionDueDate = Instant.ofEpochSecond(transaction.getHold().getDueDateTime());
+        var loanDueDate = loan.getDueDate().toInstant().truncatedTo(ChronoUnit.SECONDS);
+        if (!loanDueDate.equals(transactionDueDate)) {
+          var loanIntegerDueDate = (int) (loanDueDate.getEpochSecond() / 1000);
```

Comment:
its already seconds, what's the point to divide it by 1000?

Reply:
I didn't know. But now i know